backend/plan/views.py

```diff
@@ -399,3 +406,101 @@ def get_queryset(self):
             "sections__meetings__room",
         )
         return queryset
+
+
+class CalendarAPIView(APIView):
+    schema = PcxAutoSchema(
+        response_codes={
+            reverse_func("calendar-view", args=["schedule_pk"]): {
+                "GET": {
+                    200: "Schedule exported successfully",
+                },
+            },
+        },
+    )
+
+    def get(self, *args, **kwargs):
+        """
```

Comment:
we usually document this information in PcxAutoSchema -- see [here](https://github.com/pennlabs/penn-courses/blob/88de7109d2d2f6898719b1685baa0405131d838c/backend/PennCourses/docs_settings.py#L208) for an example.